### PR TITLE
[DOCS-6637] - ADW Developer Docs Section 5

### PR DESCRIPTION
**Pull request by Amy-Hyland**
Unable to set base as master instead of DOCS-6126

### CODE REVIEW

File: digital-workspace/latest/develop/developadfapp.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+More in particular check that you have:
+
+Alfresco Content Services (alias ACS) Enterprise edition up and running (identify the URL that will be required as configuration). 
+
+The latest lts version of NodeJs.
+
+A recent (and supported) version of a browser (see here for further details).
+
+All the Angular development is done using the Typescript language. With this in mind, it is highly suggested to adopt a good text editor to help you in this task. We recommend Visual Studio Code a free, lightweight, and very powerful tool from Microsoft that works well with Angular development and has a big ecosystem of plugins to make the developer experience even better.
```
Comment (DenysVuika) [inline digital-workspace/latest/develop/developadfapp.md:23]: I recommend turning "Visual Studio Code" block into a hyperlink with https://code.visualstudio.com/ address to help readers navigate fast.

File: digital-workspace/latest/develop/developadfapp.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    "target": "http://localhost:8080",
+    "secure": false,
+    "changeOrigin": true
+  }
+};
+```
+
+If you're running an online trial, consult your emails for the correct URL. The proxy.conf.json should look like this:
+
+module.exports = {
```
Comment (DenysVuika) [inline digital-workspace/latest/develop/developadfapp.md:83]: this block is missing backticks I guess

File: digital-workspace/latest/develop/developadfapp.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+Installing the ADF Yeoman Generator
+Note: If you're on Linux or MacOS, you might need to run the following commands using sudo.
+
+You need to ensure you have Yeoman installed by running yo --version. If this is not already installed, run the following command.
+
+```text
+npm install -g yo
+```
+
+Now install the latest version of the ``generator-alfresco-adf-app`.
```
Comment (DenysVuika) [inline digital-workspace/latest/develop/developadfapp.md:34]:
```suggestion
Now install the latest version of the `generator-alfresco-adf-app`.
```

File: digital-workspace/latest/develop/developadfapp.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+A recent (and supported) version of a browser (see here for further details).
+
+All the Angular development is done using the Typescript language. With this in mind, it is highly suggested to adopt a good text editor to help you in this task. We recommend Visual Studio Code a free, lightweight, and very powerful tool from Microsoft that works well with Angular development and has a big ecosystem of plugins to make the developer experience even better.
+
+Installing the ADF Yeoman Generator
+Note: If you're on Linux or MacOS, you might need to run the following commands using sudo.
+
+You need to ensure you have Yeoman installed by running yo --version. If this is not already installed, run the following command.
+
+```text
```
Comment (DenysVuika) [inline digital-workspace/latest/develop/developadfapp.md:30]:
```suggestion
```sh
```

File: digital-workspace/latest/develop/developadfapp.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+`npm start`
+A browser will automatically open up at http://localhost:4200.
+
+Click the key icon in the side navigation to log in.
+
+Note: If you're running an online trial, you can find your login credentials in the welcome email.
+
+Take a few minutes to explore your application. You have the essential features: Browsing the repository, uploading, and previewing documents.
+
+Troubleshooting and support
```
Comment (DenysVuika) [inline digital-workspace/latest/develop/developadfapp.md:104]: Missing heading markup

File: digital-workspace/latest/develop/developadfapp.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+If you have any issue, don’t worry! There is an entire community available to help you.
+<!-- insert links? -->
+In case of problems raise a question into the Alfresco Forum (Application Development Framework section) or connect with the developers into the Alfresco Gitter channel. Being an Alfresco customer or partner you can also raise a request for support into the Alfresco Support Portal.
+
+Other types of ADF based applications
```
Comment (DenysVuika) [inline digital-workspace/latest/develop/developadfapp.md:109]: Missing heading markup

File: digital-workspace/latest/develop/extensionmechanism.md [hunk 7]
```diff
@@ -0,0 +1,501 @@
...
+
+What you should see is a new item in “new” button on the top left of the landing page for ADW, implementing the logout from the current session. Below the screenshot describing what it should look like.
+
+Conclusions
+In this tutorial you learnt how to create your first ADW extension starting from the ADW source code stored in the GitHub repository. The purpose of this content is not to be exhaustive of all the possibilities provided by the ADW extension mechanism, but enable the developers in creating their own extensions as best practice for the customization of ADW.
+
+## How to create your first extension for the Alfresco Content Application (aka ACS)
+
+How to create your first extension for the Alfresco Content Application (aka ACA)
+Created by Francesco Corti (Deactivated)
```
Comment (DenysVuika) [inline digital-workspace/latest/develop/extensionmechanism.md:137]: the next two lines probably need to be removed

File: digital-workspace/latest/develop/extensionmechanism.md [hunk 8]
```diff
@@ -0,0 +1,501 @@
...
+NodeJs version 14.15.2,
+
+Chrome Version 87.0.4280.88.
+
+Creating the ACA extension
+As described here, the creation of an ADW extension is straightforward following the Nx Workspace dev tools for monorepos.
+
+From the root folder of the ACA project, launch the command below from a terminal. As you can see, with the command below you are going to create a new extension named my-extension.
+
+ng generate library my-extension
```
Comment (DenysVuika) [inline digital-workspace/latest/develop/extensionmechanism.md:163]: missing backticks to render as shell command

File: digital-workspace/latest/develop/extensionmechanism.md [hunk 9]
```diff
@@ -0,0 +1,501 @@
...
+In case of errors, add the following line to the tsconfig.json file.
+"compilerOptions": { "baseUrl": ".", "rootDir": "." }
```
Comment (DenysVuika) [inline digital-workspace/latest/develop/extensionmechanism.md:165]: missing backticks to render as a code snippet

File: digital-workspace/latest/develop/extensionmechanism.md [hunk 10]
```diff
@@ -0,0 +1,501 @@
...
+
+Once done, in the projects/my-extension path you will find the following structure:
+
+src folder containing all the typescript source code. Very important is the public-api.ts file defining all the inclusions of the extension and the lib/my-extension.module.ts file defining the module class for the extension.
+
+README.md file for documentation purposes as well as other files used for testing and configuration.
+
+To complete the creation, build the extension launching the following command.
+
+ng build my-extension
```
Comment (DenysVuika) [inline digital-workspace/latest/develop/extensionmechanism.md:175]: missing backticks to turn into code snippet

File: digital-workspace/latest/develop/extensionmechanism.md [hunk 11]
```diff
@@ -0,0 +1,501 @@
...
+```
+
+This is a very basic example, adding a “My Extension” item to the existing left menu, implementing a blank page containing “my-extension works!“ text appearing in the ACA landing page. From here, you can enrich the capabilities of your extension following the documentation at https://alfresco-content-app.netlify.app/#/extending/.
+
+Making the extension as part of the ACA application
+Now that the ACA extension is developed in its initial version, let's add the extension module to the list of the ones used by the application. To complete the task, edit the src/app/extensions.module.ts file as described below.
+
+// Add the following import to the page.
+
+```java
```
Comment (DenysVuika) [inline digital-workspace/latest/develop/extensionmechanism.md:242]:
```suggestion
```ts
```

File: digital-workspace/latest/develop/extensionmechanism.md [hunk 12]
```diff
@@ -0,0 +1,501 @@
...
+            ]
+        }
+    ]
+}
+```
+
+To declare the component identifier directly into the extension’s module. For that purpose edit the libs/my-extension/src/lib/my-extension.module.ts updating the following content.
+
+// Add the following imports.
+```json
```
Comment (DenysVuika) [inline digital-workspace/latest/develop/extensionmechanism.md:444]:
```suggestion
```ts
```

File: digital-workspace/latest/develop/extensionmechanism.md [hunk 13]
```diff
@@ -0,0 +1,501 @@
...
+import { ExtensionService } from '@alfresco/adf-extensions';
+import { MyFirstPageComponent } from './my-first-page/my-first-page.component';
+```
+
+// Change the NgModule as follows.
+
+```java
```
Comment (DenysVuika) [inline digital-workspace/latest/develop/extensionmechanism.md:451]:
```suggestion
```ts
```

File: digital-workspace/latest/develop/developadfapp.md (hunk 1 above)
Comment (Amy-Hyland) [reply digital-workspace/latest/develop/developadfapp.md]: Removed Microsoft VS Code reference as per another person's suggestion.

File: digital-workspace/latest/develop/extensionmechanism.md (hunk 11 above)
Comment (Amy-Hyland) [reply digital-workspace/latest/develop/extensionmechanism.md:239]: I'll confirm this change with Martin. I have made a note of it.

**Comment by Amy-Hyland** [comment]
Please do not use this PR. This does not correspond to a markdown page.

**Comment by anxumalo** [comment]
Updates completed by PR #708 

- https://github.com/Alfresco/docs-alfresco/pull/708

**Pull request closed without merging**